tools/single_use/SaveUserIDToInstrumentData.php

```diff
@@ -62,10 +62,16 @@
     $idxHist[$entry['Test_name']][] = $entry;
 }
 
-foreach (\Utility::getAllInstruments() as $testname => $fullName) {
+$instruments = \NDB_BVL_Instrument::getInstrumentNamesList($lorisInstance);
```

Comment:
this should properly be replaced with getInstrumentsList since one of the first things it does is instantiate the instrument which is already done upstream in https://github.com/aces/Loris/blob/main/php/libraries/NDB_BVL_Instrument.class.inc#L2986